aws/cloudformation/components/www_redirect.yml.erb

```diff
@@ -0,0 +1,110 @@
+  RedirectDistribution:
+    Type: AWS::CloudFront::Distribution
+    Properties:
+      DistributionConfig:
+        Enabled: true
+        Comment: Redirect www to root domain
+        PriceClass: PriceClass_All
+        Aliases:
+          - www.code.org
+        Origins:
+          # An origin is required, but will never receive traffic
+          - Id: PrimaryOrigin
+            DomainName: code.org
+            CustomOriginConfig:
+              OriginProtocolPolicy: match-viewer
+        DefaultCacheBehavior:
```

Comment:
Would be useful to set `RealtimeLogConfigArn`  To the imported `AccessLogs-Config` so we’re logging the HTTP requests before they get redirected.